chain/polkadot/keys.go

```diff
@@ -0,0 +1,118 @@
+package polkadot
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v2"
+
+	schnorrkel "github.com/ChainSafe/go-schnorrkel/1"
+	"github.com/StirlingMarketingGroup/go-namecase"
+	p2pCrypto "github.com/libp2p/go-libp2p-core/crypto"
+	"golang.org/x/crypto/blake2b"
+)
+
+const (
+	ss58Ed25519HDKDPrefix = "Ed25519HDKD"
+	ss58Secp256k1Prefix   = "Secp256k1HDKD"
+)
+
+var DEV_SEED, _ = hex.DecodeString("fac7959dbfe72f052e5a0c3c8d6530f202b02fd8f9f5ca3580ec8deb7797479e")
```

Comment:
should this be in a test file?

Reply:
This is the polkadot dev seed that is used as the base for the dev name-based key derivation

https://github.com/polkadot-js/common/blob/2d4509c32ab1bcddb85c2db16c8d40d03e85aae5/packages/keyring/src/defaults.ts#L8